wd_core.au3

```diff
@@ -88,6 +88,8 @@ Global Const $_WD_JSON_Shadow = "[value][" & $_WD_SHADOW_ID & "]"
 Global Const $_WD_JSON_Error = "[value][error]"
 Global Const $_WD_JSON_Message = "[value][message]"
 
+Global Const $JSON_MLREFORMAT = 1024 ; Addition to constants from json.au3
```

Comment:
In my opinion this variable should appear in JSON.au3 UDF and not in wd_core.au3 UDF

https://www.autoitscript.com/forum/topic/148114-a-non-strict-json-udf-jsmn/?do=findComment&comment=1517713

Reply:
I just replied to your forum post, so no point in repeating myself here.

Reply:
After I read forum answer my conclusion is to change value from 2^10 to 2^20 = 1048576